### ISSUE TITLE
rtl8723be on HP Pavilion x360 Convertible - Wifi stops working a while

### ISSUE THREAD

**Issue by doughdemon** [lwfinger/rtlwifi_new#119]
On an HP Pavilion x360 Convertible/80D1 with BIOS F.13 08/10/2015, using kernel 4.6.0-rc6 and the master branch of this repo, after loading the driver dmesg gets continuously spammed with messages

```
pcieport 0000:00:1c.4: AER: Corrected error received: id=00e4
pcieport 0000:00:1c.4: PCIe Bus Error: severity=Corrected, type=Physical Layer, id=00e4(Receiver ID)
pcieport 0000:00:1c.4:   device [8086:9d14] error status/mask=00000001/00002000
pcieport 0000:00:1c.4:    [ 0] Receiver Error
```

while wifi is working fine. After some time the connection drops,

```
rtlwifi:rtl_watchdog_wq_callback():<0-0> AP off, try to reconnect now
wlan0: Connection to AP a0:1b:29:b6:81:20 lost
rtl8723be:rtl8723be_hw_init():<0-0> Init MAC failed
```

can be found in dmesg and a reboot seems necessary to make it work again.

Before starting to use "ant_sel=1" the signal strength was very bad but now it is fine. The setting of "ips" does not seem to make any qualitative difference. (It might shorten the time of wifi being usable, though.) I have not touched the parameter "msi".

I would be glad for any pointers on how this can be fixed and am happy to provide further data.

**Comment by izbushka** [comment]
> AER: Corrected error received: id=00e4
This has nothing to do with WiFi
You can silence those errors as described here: AER: https://lkml.org/lkml/2015/9/3/475

As for the wifi problem I can't help, sorry

**Comment by doughdemon** [comment]
> You can silence those errors as described here: AER: https://lkml.org/lkml/2015/9/3/475

Thanks for the pointer! (I haven't tried it out yet since to me
silencing those messages has much lower priority than working WiFi.)

**Comment by german-rios-gonzalez** [comment]
Hi, to correct the wifi drop after a while set fwlps=0 the same way like you do with "ant_sel=1"

**Comment by lwfinger** [comment]
A completely new version of the drivers has just been pushed. It fixes a lot of things. As a result, I am closing all open issues. If you get a new one, please create a new issue.

Note: The new driver fixes the problems with power save. It is recommended that you clear all ips, swlps, and fwlps options.
